modules/RTC/TraceablePeerConnection.js

```diff
@@ -2668,6 +2633,8 @@ TraceablePeerConnection.prototype.setSenderVideoConstraints = function(frameHeig
                     .find(layer => layer.scaleResolutionDownBy === scaleFactor)?.maxBitrate ?? bitrate;
             }
             parameters.encodings[0].maxBitrate = bitrate;
+        } else {
+            parameters.encodings[0].maxBitrate = undefined;
```

Comment:
would 0 work here? In RN the native APIs only support a number or null.

Reply:
We want the maxBitrate to be not limited here for VP9-p2p case so 0 will not work.

Reply:
Got it. We might need to adjust RN I think.